src/isomorphic/classic/class/ReactClass.js

```diff
@@ -73,15 +71,15 @@ var SpecPolicy = keyMirror({
  * @interface ReactClassInterface
  * @internal
  */
-var ReactClassInterface = {
+var ReactClassInterface: {[key: string]: SpecPolicy} = {
```

Comment:
The file is not @flow so it's not doing anything but prevents eslint from thinking that SpecPolicy is unused.